RealEstate/RealEstate/Pages/Index.cshtml.cs

```diff
@@ -32,11 +32,23 @@ public void OnGet()
                 // Convert raw text to list.
                 List<Houses> houseCollection = Houses.FromJson(houseJSON);
 
-
                 ViewData["Houses"] = houseCollection;
 
                 var landJSON = webClient.DownloadString("https://data.cityofchicago.org/resource/aksk-kvfp.json");
 
+                
+                string result = "";
+
+                try
+                {
+                    result = webClient.DownloadString(landJSON);
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine("Exception while calling the Land API", e);
+
+               }
```

Comment:
Logging is a good idea, but what should you do with the exception at this point?  Re-throw it?  If you just log it and bury it, the program will continue to execute after the exception occurs, as if nothing went wrong.  That is worse than the default behavior: terminate the method when an exception occurs.

Use proper logging instead of Console.WriteLine().
- Logging has the concept of priority (error, debug, warn, etc.), which Console.WriteLine() doesn't have.
- No one will see Console.WriteLine(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with Console.WriteLine().